src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -57,6 +57,13 @@ private void OnPreviewMouseWheel(object sender, MouseWheelEventArgs e)
             }
```

Comment:
Hi @aosyatnik, if you could update the comments as follow, that will be great:

```csharp
// Case 1: User scrolls the mouse wheel when Shift key is being held down. This 
// action triggers a horizontal scroll on the library view (so that lengthy names 
// can be revealed). Setting 'Handled' to 'false' allows the underlying scroll bar
// to handle the mouse wheel event.

// Case 2: If the mouse is outside of the library view, but mouse wheel messages 
// get sent to it anyway. In such case there is nothing to change here. The 'Handled'
// is not set to 'true' here because the mouse wheel messages should be routed to the 
// ScrollViewer on tool-tip for further processing.

// Case 3: 
// Mouse wheel without any modifier keys, it scrolls the library view vertically. In 
// this case 'VerticalOffset' is updated, 'Handled' is also set so that mouse wheel 
// message routing ends here.
```